src/test-run/bookmark.js

```diff
@@ -43,7 +45,12 @@ class TestRunBookmark {
                                             new SwitchToIframeCommand({ selector: this.iframeSelector }) :
                                             new SwitchToMainWindowCommand();
 
-            await this.testRun.executeCommand(switchWorkingFrameCommand);
+            try {
+                await this.testRun.executeCommand(switchWorkingFrameCommand);
```

Comment:
Actually [a few types of errors ](https://github.com/DevExpress/testcafe/blob/d420397f895edb9bd75d788ef729347128057686/src/client/driver/driver.js#L62)can be raised during execution of switching command